web/Puzzle.tsx

```diff
@@ -18,11 +18,13 @@ const disableShuffle = TESTING_MODE;
 
 export default function Puzzle({
   puzzle,
+  startSolved,
 }: {
   puzzle: PuzzleType;
+  startSolved: boolean;
 }): JSX.Element {
   const [height, setHeight] = useState(0);
-  const [solved, setSolved] = useState(false);
+  const [solved, setSolved] = useState(startSolved);
   const [pieces, setPieces] = useState<Piece[]>([]);
   const [snapPoints, setSnapPoints] = useState<Point[]>([]);
   const boardRef = useRef<HTMLDivElement | null>(null);
```

Comment:
Hey, so I know you didn't add this in this PR, but since you have the PR open anyways and are editing the file, how do you feel about removing the console.log on Line 43?